buildSrc/src/main/groovy/Dependencies.groovy

```diff
@@ -16,6 +16,8 @@ class Dependencies {
   static def springBootStarterWeb = { it=Versions.springBoot -> "org.springframework.boot:spring-boot-starter-web:${it}" }
   static def springDataSolr = { it=Versions.springData -> "org.springframework.data:spring-data-solr:${it}" }
   static def transformAPI = { it=Versions.transformAPI -> "com.connexta.transformation:transformation-api-rest-spring-stubs:${it}" }
+  static def springActuator = { it=Versions.springBoot -> "org.springframework.boot:spring-boot-starter-actuator:${it}" }
```

Comment:
❓ I wonder if we need to lock this endpoint now at this point.

Reply:
Eventually, yes. It is not something you want exposed in a production system. We can do that Until we integrate with the security services.